app/assets/javascripts/hotwire_spark.js

```diff
@@ -1569,6 +1569,10 @@ var HotwireSpark = (function () {
       return new ReplaceHtmlReloader().reload();
     }
     async reload() {
+      if (!window.Turbo) {
+        console.log(`[hotwire-spark] Tried to replace the page with Turbo, but Turbo is not available on window.Turbo`);
```

Comment:
A small nitpick to align with the existing logger:

```suggestion
        console.log(`[hotwire_spark] Tried to replace the page with Turbo, but Turbo is not available on window.Turbo`);
```

I'd probably extract to `logger.js`  and extract a `const TAG = "[hotwire_spark]"`.